controllers/games-controllers.js

```diff
@@ -0,0 +1,11 @@
+const {selectCategories} = require('../models/games-models');
+
+const getCategories = (req, res, next) => {
+  selectCategories().then((response) => {
```

Comment:
We have control over this parameter name, so something like categories would be more descriptive.